gulpfile.js

```diff
@@ -6,6 +6,8 @@ var ON_DEATH = require('death')({ debug: false });
 var watch = require("gulp-watch");
 var del = require('del');
 var sass = require('gulp-sass');
+var shell = require('shelljs');
```

Comment:
How is shelljs exec different from child-process exec?

Why use this rather than the builtins for _dirname and process.chdir?

Reply:
I had loads of trouble convincing the built in child-process exec running ng build command in the correct cwd and not inheriting stuff from the seed project build. The only exec I could to get to work (from the handful I tried) was this, which is actually in use in the CLI, where its used to run the project generation commands.

Can re-attempt with the built in versions, but was very happy when did not return the various avalanches of digital vomit I was getting out of other solutions.